test/jpypetest/common.py

```diff
@@ -103,6 +104,7 @@ def setUp(self):
 
             classpath_arg %= jpype.getClassPath()
             args.append(classpath_arg)
+            _jpype.enableStacktraces(True)
```

Comment:
Is "enableStacktraces" a hidden feature, we should advertise in the documentation? I guess it becomes handy during all kind of debug scenarios.

Reply:
Yes.  It is an internal feature documented in the section "Getting additional diagnostics" along with a handful of developer features that I use for debugging.  

https://jpype.readthedocs.io/en/latest/userguide.html#c-exceptions-in-jpype

The only section in that area I am aware is currently missing are tricks to use jlink and debugging threading issues.

Reply:
That addition sounds interesting. The guide is already super useful! Thank you

Reply:
I am trying set a new record for largest documentation to user base ratio.  Helps that denominator is small. :wink: